ui/components/component-library/button-icon/deprecated/button-icon.test.js

```diff
@@ -1,8 +1,8 @@
 /* eslint-disable jest/require-top-level-describe */
 import { render } from '@testing-library/react';
 import React from 'react';
-import { IconColor } from '../../../helpers/constants/design-system';
-import { IconName } from '..';
+import { IconColor } from '../../../../helpers/constants/design-system';
+import { ICON_NAMES } from '../../icon/deprecated';
 import { BUTTON_ICON_SIZES } from './button-icon.constants';
```

Comment:
The changes look really good to me. Before I approve it, one quick question - Should we update BUTTON_ICON_SIZES as well since this is a part of ButtonIcon??